src/Modules/OrchardCore.Commerce/Serialization/LegacyShoppingCartItemConverter.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using OrchardCore.Commerce.Abstractions;
+using OrchardCore.Commerce.Models;
+using OrchardCore.Commerce.ProductAttributeValues;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OrchardCore.Commerce.Serialization;
+
+internal sealed class LegacyShoppingCartItemConverter : JsonConverter<ShoppingCartItem>
+{
+    public override void WriteJson(JsonWriter writer, ShoppingCartItem value, JsonSerializer serializer)
+    {
+        writer.WriteStartObject();
+
+        writer.WritePropertyName(ShoppingCartItemConverter.QuantityName);
+        writer.WriteValue(value.Quantity);
+
+        writer.WritePropertyName(ShoppingCartItemConverter.SkuName);
+        writer.WriteValue(value.ProductSku);
+
+        if (value.Prices?.Any() == true)
+        {
+            writer.WritePropertyName(ShoppingCartItemConverter.PricesName);
+            serializer.Serialize(writer, value.Prices);
+        }
+
+        if (value.Attributes != null && value.Attributes.Any())
```

Comment:
This too could be:
`value.Prices?.Any() == true`

https://github.com/OrchardCMS/OrchardCore.Commerce/pull/330#discussion_r1306442721

Reply:
@DAud-IcI Sorry it's `value.Attributes` not `Prices` here.